### DISCUSSION TITLE
blog/changelog/

### DISCUSSION THREAD

**Discussion by giscus** [darstib/blog#2] [General]
# blog/changelog/

Darstib's blog!

https://darstib.github.io/blog/changelog/

**Comment by darstib** [comment]
参考 [xuan-insr 的设置](https://github.com/xuan-insr/xuan-insr.github.io) ，成功创建好了评论功能，欢迎留言交换友链，或者指出对应文章的不当之处哦。

欢迎留言交换友链！

交换友链请参考以下格式：
- `主页 url` https://darstib.github.io/
- `[opt]: 昵称` = Darstib
- `[opt] 头像 url` = https://darstib.github.io/pic/avatar.jpg
- `[opt] 一句话` = Welcome To Darstib's little world !

> `opt` 表示可选，我从 `主页 url` 那里自己找咯~
> 一句话希望不要太长😢，css比较菜，控制不住爆出来了就不太好看了……
